src/codegen/templates/system/physical_channel.py.mako

```diff
@@ -34,12 +34,14 @@ class PhysicalChannel:
     """
     __slots__ = ['_name', '__weakref__']
 
-    def __init__(self, name):
+    def __init__(self, name, interpreter):
```

Comment:
This is a public constructor. Adding a required `interpreter` parameter breaks compatibility with the example code in README.rst:
```
>>> phys_chan = device.ai_physical_chans['ai0']
>>> phys_chan
PhysicalChannel(name=Dev1/ai0)
>>> phys_chan == nidaqmx.system.PhysicalChannel('Dev1/ai0')
True
```

Instead, this class should take an optional, keyword-only `grpc_options`, and you should create a `_PhysicalChannelAlternateConstructor` to allow other code to inject an interpreter. See my suggestion on `scale.py.mako`.